src/utils/declareSsrProps.ts

```diff
@@ -26,7 +23,7 @@ export interface ExternalPageProps<P = { [key: string]: string }> extends SSRPro
 }
 
 export function declareSsrProps<T = ExternalPageProps>(
-    cb?: ({ user, locale, req, params, query }: SSRProps) => T,
```

Comment:
I think this is breaking changes for many places in app which code based on https://github.com/taskany-inc/issues/blob/main/src/components/Page.tsx#L66

Reply:
fixed 'em all